src/platforms/ios/internal/NativeScript.framework/Headers/NativeScript.h

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
I don't think we need to version all of that. Only the `objc!TwilioVoice.d.ts` is enough.

Reply:
Agreed 💯 removed the src/platforms/ios/internal/ directory